include/player.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __PLAYER_H
+#define __PLAYER_H
+
+#include "ledRunner.h"
+
+class Player {
+    private:
+        int startLed;
+        int endLed;
+        int ledState;
+        ledMode_t ledMode;
+        CRGB colour;
+    public:
+        Player(int startLed, int endLed, CRGB colour);
+        virtual ~Player();
+        virtual void setLedMode(ledMode_t mode);
+};
+
+#endif /* __PLAYER_H */
```

Comment:
Hold per-player info - where their LEDs are in the global array, what colour the player is, what mode their LEDs are running in, etc